packages/suite/src/components/wallet/WalletLayout/AccountTopPanel/AccountNavigation.tsx

```diff
@@ -50,6 +41,7 @@ export const AccountNavigation = () => {
             },
             title: <Translation id="TR_NAV_TRANSACTIONS" />,
             isHidden: false,
+            isTabActive: ['wallet-index'],
```

Comment:
instead of definining this for each route. 
I would rather do:
- if not specified, use id
- if specified, use isTabActive

Reply:
And why you don't like this solution? I think it's fine no?